ControlRoomApplication/ControlRoomApplication/Controllers/Communications/RemoteListener.cs

```diff
@@ -128,188 +217,619 @@ public bool RequestToKillTCPMonitoringRoutine()
-                config.AzimuthTemp1Init = splitData[(int)SensorInitializationEnum.AzimuthTemp].Equals("1");
-                config.ElevationTemp1Init = splitData[(int)SensorInitializationEnum.ElevationTemp].Equals("1");
-                config.AzimuthAccelerometerInit = splitData[(int)SensorInitializationEnum.AzimuthAccelerometer].Equals("1");
-                config.ElevationAccelerometerInit = splitData[(int)SensorInitializationEnum.ElevationAccelerometer].Equals("1");
-                config.CounterbalanceAccelerometerInit = splitData[(int)SensorInitializationEnum.CounterbalanceAccelerometer].Equals("1");
-                config.AzimuthEncoderInit = splitData[(int)SensorInitializationEnum.AzimuthEncoder].Equals("1");
-                config.ElevationEncoderInit = splitData[(int)SensorInitializationEnum.ElevationEncoder].Equals("1");
+                   
+                case "REQUEST":
+                    if (splitCommandString.Length != TCPCommunicationConstants.NUM_REQUEST_PARAMS)
+                    {
+                        return new ParseTCPCommandResult(ParseTCPCommandResultEnum.MissingCommandArgs, splitCommandString, TCPCommunicationConstants.MISSING_COMMAND_ARGS + command);
+                    }
 
-                // Set the timeout values
-                config.TimeoutDataRetrieval = (int)(double.Parse(splitData[7]) * 1000);
-                config.TimeoutInitialization = (int)(double.Parse(splitData[8]) * 1000);
+                    switch (splitCommandString[TCPCommunicationConstants.REQUEST_TYPE])
+                    {
+                        case "MVMT_DATA":
+                            return new ParseTCPCommandResult(ParseTCPCommandResultEnum.Success, splitCommandString);
 
-                // Reboot
-                rtController.RadioTelescope.SensorNetworkServer.RebootSensorNetwork();
+                        default:
+                            return new ParseTCPCommandResult(ParseTCPCommandResultEnum.InvalidRequestType, splitCommandString, TCPCommunicationConstants.INVALID_REQUEST_TYPE + splitCommandString[TCPCommunicationConstants.REQUEST_TYPE]); ;
+                    }
+                    break;
 
-                return true;
+                // if we get here, command type not found
+                default:
+                    return new ParseTCPCommandResult(ParseTCPCommandResultEnum.InvalidCommandType, splitCommandString, TCPCommunicationConstants.COMMAND_NOT_FOUND);
+              
             }
 
-            // can't find a keyword then we fail
-            return false;
+
+            // else all parsing was successful, return and inform client
+            return new ParseTCPCommandResult(ParseTCPCommandResultEnum.Success, splitCommandString);
+        }
+
+            /// <summary>
+            /// Util method to handle error checking with strem.read/writes
+            /// </summary>
+            /// <param name="text">What you are sending back to the client</param>
```

Comment:
These documentation comments should be indented 1 tab back to line up with the method call.

- [ ] Consider unindenting documentation comments